agent/command/test_selection_get_test.go

```diff
@@ -0,0 +1,158 @@
+package command
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/evergreen-ci/evergreen/agent/internal"
+	"github.com/evergreen-ci/evergreen/agent/internal/client"
+	"github.com/evergreen-ci/evergreen/model"
+	"github.com/evergreen-ci/evergreen/model/task"
+	"github.com/evergreen-ci/utility"
+	"github.com/stretchr/testify/suite"
+)
+
+type TestSelectionGetSuite struct {
```

Comment:
I don't think we should use a testify suite for these tests since they are brand new ([doc](https://docs.google.com/document/d/1-DB1FQ_zb5fX_0pnVcwyU0HmJPrloFBu6qSE8dkk8PU/edit?tab=t.0#heading=h.kgvwy8q09axn))